elastic/kibana
# [Maps] fix Cannot open Lens editor in airgapped environment

Fixes https://github.com/elastic/kibana/issues/132408

PR wraps all calls to `getEmsFileLayers` and `getEmsTmsServices` with try/catch block. These calls throw when EMS is unavailable.

To test
* Start kibana server
* Disconnect computer from network
* Open Kibana in browser and create new lens visualization. Lens should open and regionmap chart will provide disabled "EMS boundaries" select with message about being unable to access EMS and no suggestions for regionmap chart will be provided.

<img width="500" alt="Screen Shot 2022-05-18 at 7 15 28 AM" src="https://user-images.githubusercontent.com/373691/169052609-909ba6a9-ad86-4d62-abc0-78cd05bc1b90.png">
